specification/schemas/shops/Shop.json

```
@@ -132,14 +132,22 @@
             "locale": {
               "allOf": [
                 {
-                  "$ref": "#/components/schemas/Locale"
+                  "$ref": "#/components/schemas/LocaleEnum"
                 },
                 {
                   "default": "en-GB",
                   "example": "nl-NL",
                   "description": "The default language that is used for emails and webpages for this shop."
                 }
               ]
+            },
+            "return_portal_locales": {
+              "type": "array",
+              "items": {
+                "$ref": "#/components/schemas/LocaleEnum"
+              },
+              "example": "nl-NL",
```

Example appears to be wrong, should be an array
```suggestion
              "example": ["nl-NL"],
```

---

fixed